runtime/Makefile

```diff
@@ -14,7 +14,7 @@ ENABLE_LIGHTNING?=ON
 ENABLE_LIGHTNING_KOKKOS?=ON
 ENABLE_OPENQASM?=ON
 ENABLE_ASAN?=OFF
-LIGHTNING_GIT_TAG_VALUE?=latest_release
+LIGHTNING_GIT_TAG_VALUE?=seed_sample_lightning
```

Comment:
wait for the lightning PR to be merged first, then track that commit

Reply:
https://github.com/PennyLaneAI/pennylane-lightning/commit/6f3e0d5d371ff9823a3177dd2c66052668883d42